packages/waku/src/lib/middleware/dev-server-impl.ts

```diff
@@ -105,7 +104,6 @@ const createMainViteServer = (
           cacheDir: 'node_modules/.vite/waku-dev-server-main',
           base: config.basePath,
           plugins: [
-            patchReactRefresh(viteReact()),
```

Comment:
Did you mean to remove `viteReact()` entirely?

Reply:
Oh, noooo.

Reply:
Thanks. What a shame...